docs/developer-experience/artificial-intelligence.md

```diff
@@ -0,0 +1,63 @@
+# Artificial Intelligence
```

Comment:
I feel the title is misleading, I think it needs to be Copilots and not AI as AI is a big topic 
```suggestion
# Copilots
```